core/chainlink.cre.Dockerfile

```
@@ -0,0 +1,98 @@
+##
+# Build image: Chainlink binary with plugins.
+##
+FROM golang:1.24-bullseye AS buildgo
+RUN go version
+RUN apt-get update && apt-get install -y jq && rm -rf /var/lib/apt/lists/*
+
+WORKDIR /chainlink
+
+COPY GNUmakefile package.json ./
+COPY tools/bin/ldflags ./tools/bin/
+
+ADD go.mod go.sum ./
+RUN --mount=type=cache,target=/go/pkg/mod \
+    go mod download
+COPY . .
+
+# Install Delve for debugging with cache mounts
+RUN --mount=type=cache,target=/go/pkg/mod \
+    --mount=type=cache,target=/root/.cache/go-build \
+    go install github.com/go-delve/delve/cmd/dlv@v1.24.2
+
+# Flag to control installation of private plugins (default: false).
+ARG CL_INSTALL_PRIVATE_PLUGINS=false
+# Flag to control installation of testing plugins (default: false).
+ARG CL_INSTALL_TESTING_PLUGINS=false
+# Flags for Go Delve debugger
+ARG GO_GCFLAGS
+# Env vars needed for chainlink build
+ARG COMMIT_SHA
+
+ENV CL_LOOPINSTALL_OUTPUT_DIR=/tmp/loopinstall-output
+RUN --mount=type=secret,id=GIT_AUTH_TOKEN \
+    --mount=type=cache,target=/go/pkg/mod \
+    --mount=type=cache,target=/root/.cache/go-build \
+    ./plugins/scripts/setup_git_auth.sh && \
+    mkdir -p /gobins && mkdir -p "${CL_LOOPINSTALL_OUTPUT_DIR}" && \
+    GOBIN=/go/bin make install-loopinstall && \
+    GOBIN=/gobins CL_LOOPINSTALL_OUTPUT_DIR=${CL_LOOPINSTALL_OUTPUT_DIR} make install-cre-plugins-local install-cre-plugins-public && \
+    if [ "${CL_INSTALL_PRIVATE_PLUGINS}" = "true" ]; then \
+        GOBIN=/gobins CL_LOOPINSTALL_OUTPUT_DIR=${CL_LOOPINSTALL_OUTPUT_DIR} make install-cre-plugins-private; \
+    fi && \
+    if [ "${CL_INSTALL_TESTING_PLUGINS}" = "true" ]; then \
+        GOBIN=/gobins CL_LOOPINSTALL_OUTPUT_DIR=${CL_LOOPINSTALL_OUTPUT_DIR} make install-cre-plugins-testing; \
+    fi
+
+# Copy any shared libraries.
+RUN --mount=type=cache,target=/go/pkg/mod \
+    mkdir -p /tmp/lib && \
+    ./plugins/scripts/copy_loopinstall_libs.sh \
+    "$CL_LOOPINSTALL_OUTPUT_DIR" \
+    /tmp/lib
+
+# Build chainlink.
+RUN --mount=type=cache,target=/go/pkg/mod \
+    --mount=type=cache,target=/root/.cache/go-build \
+    GOBIN=/gobins make GO_GCFLAGS="${GO_GCFLAGS}" install-chainlink
```

```suggestion
    GOBIN=/gobins make install-chainlink
```

If we avoid installing delve.